### PR TITLE
Add the Maven Wrapper to Presto

### PR DESCRIPTION
**Pull request by jvanzyl**
This will allow Presto to transparently use newer versions of Maven, and transparently allow
users to take advantage of those features without any burden on their part.

This will not affect any current users and this feature can bake as the
documentation is updated on the Presto site and is seen by users. I
recommend leaving the wrapper in the build for a few weeks before
adding any features that require its use.

More information about the wrapper can be found here:

https://github.com/takari/maven-wrapper

### CODE REVIEW

File: .mvn/wrapper/maven-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+distributionUrl=https://repo1.maven.org/maven2/org/apache/maven/apache-maven/3.3.3/apache-maven-3.3.3-bin.zip
```
Comment (electrum) [inline .mvn/wrapper/maven-wrapper.properties]: Add a newline

**Comment by electrum** [comment]
Let's remove the Windows script for now since we don't support building on Windows

File: .mvn/wrapper/maven-wrapper.properties (hunk 1 above)
Comment (martint) [inline .mvn/wrapper/maven-wrapper.properties]: Why is this hard-coded? Can't we use the config in .m2/settings.xml?

Comment (jvanzyl) [inline .mvn/wrapper/maven-wrapper.properties]: It's intended to be project specific, so there's a fixed value of a known version of Maven that works with the particular build in question. If you wanted a URL that is to be used across an organization that would have to be added. Right now it's only meant to be setup on a per project basis.

We can also set additional project specific settings like JVM memory settings, extensions to use and default maven flags to use.

**Pull request closed without merging**